conf/src/lib.rs

```diff
@@ -234,6 +246,27 @@ mod tests {
         cleanup_file(path.as_str());
     }
 
+    #[test]
+    fn flush_conf_three() {
+        let path = get_conf_path();
```

Comment:
Can we test the case where we want remove all the plugins, and the case when we want remove just a single plugin?

Reply:
Ok I added another test for removing just one plugin